app/templates/public/index.hbs

```diff
@@ -11,7 +11,7 @@
         <div class="ui segment">
           <h3 class="ui header">{{t 'Tickets'}}</h3>
         </div>
-        <div class="ui segment">
+        <div class="ui segment x-scrollable">
```

Comment:
https://github.com/fossasia/open-event-frontend/pull/3583/files
same chage was made by me in this pr, but seems some-one reverted it.
@snitin315 , i hope you wanna see it.

Reply:
@kushthedude can you share the reason for reverting?

Reply:
I am also not able to understand, it was working quite well :confused:

Reply:
I guess it's not if there is only one ticket available to buy. (In desktop view) when we click on ticket dropdown it gets weird.

Reply:
Are you sure that weirdness is due to `x-scrollable` ?

Reply:
Yes ,
I would have shared screenshot but i am working on a different branch right now.

Reply:
We have make it scrollable only for mobile devices. Rest is all fine in my opinion.

Reply:
Don't use DOM, IMO.